sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/LinkHandler.java

```diff
@@ -88,7 +91,8 @@ private void handleRemoteLinkClosed(final String eventName, final Event event) {
 
             onError(exception);
         } else {
-            close();
+            onError(new AmqpException(true, String.format("This link is closed by %s without a condition.", eventName),
```

Comment:
This change is to let the exception handling and link processor retry to kick in.